src/test/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,29 @@
+package calculator;
+
+public class StringCalculator {
+    private static final int DEFAULT_RESULT = 0;
+
+    private final Separator separator;
+
+    public StringCalculator() {
+        this(new TextSeparator());
+    }
+
+    public StringCalculator(Separator separator) {
+        this.separator = separator;
+    }
+
+    public int add(String text) {
+        if (isEmptyText(text)) {
+            return DEFAULT_RESULT;
+        }
+
+        return separator.separate(text).stream()
+                .mapToInt(Integer::valueOf)
+                .sum();
+    }
+
+    private boolean isEmptyText(String text) {
+        return text == null || text.isEmpty();
```

Comment:
```suggestion
        return StringUtils.hasText(text);
```

spring 에서 제공해주는 StringUtils 를 활용해보는 것도 좋아요~

Reply:
사실 StringUtils를 사용할까 고민 했었습니다! 
하지만 문자열 덧셈 계산기 같은 경우 Spring이 필요 없는 미션이라고 생각 되어서 사용하지 않았습니다 😭
이 부분에 관해서 리뷰어님은 어떻게 생각하실까요?

Reply:
현재 ddd-legacy 프로젝트가 스프링으로 되어있기 때문에 전 사용해도 좋을 것 같아요 👍 
이 부분은 개발자 분들 마다 다를 수도 있겠네요 😭